packages/moonstone/Scrollable/ScrollButton.js

```diff
@@ -81,6 +81,8 @@ const ScrollButton = kind({
 	}
 });
 
+const ScrollButton = React.forwardRef((props, ref) => <ScrollButtonBase {...props} ref={ref} />);
```

Comment:
Hmm.. A big advantage of using `forwardRef` is to encapsulate a component so that it would be independent from other components' structure. So, I think it may be better to move `forwardRef` code to a consumer component; `ScrollButtons` in our code.

Reply:
That's good idea.